base/applications/cmdutils/more/more.c

```diff
@@ -529,12 +951,9 @@ int wmain(int argc, WCHAR* argv[])
     /* Load the registry settings */
     LoadRegistrySettings(HKEY_LOCAL_MACHINE);
     LoadRegistrySettings(HKEY_CURRENT_USER);
+    if (bEnableExtensions)
+        s_dwFlags |= FLAG_E;
 
-    // TODO: First, load the "MORE" environment variable and parse it,
-    // then parse the command-line parameters.
-
-    // FIXME: Parse all the remaining parameters.
-    // Then the file list can be found at the very end.
     // FIXME2: Use the PARSER api that can be found in EVENTCREATE.
```

Comment:
Regarding that FIXME2: don't touch it, just keep the current command-line parsing as you are doing here.

Reply:
OK.